halo2_proofs/src/poly/domain.rs

```diff
@@ -245,7 +264,17 @@ impl<F: WithSmallOrderMulGroup<3>> EvaluationDomain<F> {
 
         self.distribute_powers_zeta(&mut a.values, true);
         a.values.resize(self.extended_len(), F::ZERO);
-        best_fft(&mut a.values, self.extended_omega, self.extended_k);
+
+        let l = a.len();
+        let fft_data = self.get_fft_data(l);
+
+        best_fft(
+            &mut a.values,
+            self.extended_omega,
+            self.extended_k,
+            fft_data,
+            false,
+        );
```

Comment:
```suggestion
        best_fft(
            &mut a.values,
            self.extended_omega,
            self.extended_k,
            &self.extended_fft_data
            false,
        );
```

Always extended anyway.

Reply:
You are right.  I  made the field `pub`.